.github/workflows/go.yaml

```diff
@@ -1,17 +1,17 @@
 name: Go
 on:
   push:
-    paths-ignore:
-      - "docs/**"
-      - "webui/**"
-      - "design/**"
+#    paths-ignore:
```

Comment:
What are mandatory tests? Do all 3 contain these?

Reply:
our go tests are required to pass before merging to master
paths-ignore means that if I only changed files under `docs/` or `webui/`, the go tests will not run.
This is a deadlock - I'll never be able to merge my changes because mandatory tests are conditional.